packages/web/src/components/payout-wallet-modal/PayoutWalletModal.tsx

```diff
@@ -148,26 +153,53 @@ export const PayoutWalletModal = () => {
   }, [setIsOpen])
 
   const handleSubmit = useCallback(
-    ({ option, address }: PayoutWalletValues) => {
-      // TODO: Token account validation
-      if (user) {
+    async (
+      { option, address }: PayoutWalletValues,
+      { setErrors }: { setErrors: any }
+    ) => {
+      try {
+        if (!address || !user) {
+          throw new Error('No user or address')
+        }
+
+        const isUsdcAta = await isTokenAccount({
```

Comment:
is this enough? what if the address is a WAUDIO token account? we should probably not create the ATA then either. Maybe we want to check if it's _any_ kind of token account, _then_ check for the mint?

Reply:
agreed. let's pair on this today, will merge PR first